standup/status/models.py

```diff
@@ -12,7 +12,13 @@ class Project(models.Model):
     name = models.CharField(max_length=100, unique=True)
     slug = models.CharField(max_length=100, unique=True)
     color = models.CharField(max_length=6)
-    repo_url = models.URLField()
+    repo_url = models.URLField(max_length=100)
+
+    class Meta:
+        db_table = 'project'
+
+    def __str__(self):
+        return self.name
```

Comment:
Feel free to fix all my `__str__` things. I can't remember what Django does with them, so they're all probably wrong.